Resources/Prototypes/Recipes/Crafting/crates.yml

```diff
@@ -7,5 +7,26 @@
   category: Storage
   description: "A wooden crate for holding livestock"
   icon: Structures/Storage/Crates/livestock.rsi/livestockcrate.png
-  objectType: Item
-  
\ No newline at end of file
+  objectType: Structure
+  
+- type: construction
+  name: metal crate
+  id: CrateGenericonimo
+  graph: CrateGenericonimo
```

Comment:
genericonimo?

Reply:
Apparently thats the ID for the generic crate, beyond me but I can change it throughout if that's better.